### ISSUE TITLE
Asset precompilation fails with a freeze or unknown file, packs-test/manifest.json

### ISSUE THREAD

**Issue by kpheasey** [rails/webpacker#510]
I'm unable to precompile assets for my test environment.  Running the following `bin/rails assets:clobber assets:precompile` will cause webpacker to freeze when compiling (i've left it for over an hour).  If I use `bin/rails assets:precompile assets:clean` instead, it will fail because of an unknown file:

```
[Webpacker] Compiling assets 🎉
[Webpacker] Compiled digests for all packs in /home/ubuntu/app/app/javascript/packs:
rails aborted!
Errno::ENOENT: No such file or directory @ rb_sysopen - /home/ubuntu/app/public/packs-test/manifest.json
/home/ubuntu/app/vendor/bundle/ruby/2.3.0/bundler/gems/webpacker-a0c1e9a7f2f3/lib/tasks/webpacker/compile.rake:17:in `read'
/home/ubuntu/app/vendor/bundle/ruby/2.3.0/bundler/gems/webpacker-a0c1e9a7f2f3/lib/tasks/webpacker/compile.rake:17:in `block (2 levels) in <top (required)>'
/home/ubuntu/app/vendor/bundle/ruby/2.3.0/bundler/gems/webpacker-a0c1e9a7f2f3/lib/tasks/webpacker/compile.rake:34:in `block in <top (required)>'
/home/ubuntu/app/vendor/bundle/ruby/2.3.0/gems/railties-5.0.3/lib/rails/commands/rake_proxy.rb:14:in `block in run_rake_task'
/home/ubuntu/app/vendor/bundle/ruby/2.3.0/gems/railties-5.0.3/lib/rails/commands/rake_proxy.rb:11:in `run_rake_task'
/home/ubuntu/app/vendor/bundle/ruby/2.3.0/gems/railties-5.0.3/lib/rails/commands/commands_tasks.rb:51:in `run_command!'
/home/ubuntu/app/vendor/bundle/ruby/2.3.0/gems/railties-5.0.3/lib/rails/commands.rb:18:in `<top (required)>'
```

I

**Comment by gauravtiwari** [comment]
@kpheasey Ahh I see - could you please try - `bin/rails webpacker:clobber webpacker:compile` before running tests.

**Comment by gauravtiwari** [comment]
BTW webpacker does lazy compiling of assets in test env so you don't need to do that manually before running tests - https://github.com/rails/webpacker/blob/master/lib/webpacker/manifest.rb#L19

**Comment by kpheasey** [comment]
It must have been a configuration issue.  We implemented webpacker before Rails 5.1 release.  Everything started working after using the 2.0 gem and overwriting the install files with defaults .

We have a large project and precompile and cache our assets in CI.
